.github/workflows/main.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         env:
           REPLICATED_API_ORIGIN: https://api.replicated.com/vendor
           REPLICATED_ID_ORIGIN: https://id.replicated.com
-          VENDOR_USER_EMAIL: dexter@replicated.com
+          VENDOR_USER_EMAIL: craig+replicatedcli@replicated.com
```

Comment:
While I agree it shouldn't be `dexter@`, is there a bot account we can make/create?